File: src/main/kotlin/com/albatros/springsecurity/data/model/dto/TenderDto.java
package com.albatros.springsecurity.data.model.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class TenderDto {
    private int ID;
    private String Date;
    private String TenderName;
    private String Customer;
    private String Category;
    private String Region;
    private Long Price;
    private String EndTime;
    private String Etp;
    private String TenderLink;
    private String TenderLinkInner;
    @JsonProperty("User_id")
    private int UserId;
    private String CustomerINN;
}

File: src/main/kotlin/com/albatros/springsecurity/data/model/dto/TenderDto.kt
package com.albatros.springsecurity.data.model.dto

data class TenderDtoItem(
    val ApiTenderInfo: String,
    val Category: String,
    val Customer: String,
    val Date: String,
    val EndTime: String,
    val Etp: String,
    val Fz: String,
    val ID: String,
    val Price: String,
    val Region: String,
    val TenderLink: String,
    val TenderLinkInner: String,
    val TenderName: String,
    val TenderNumOuter: String,
    val Total: String,
    val User_id: String,
    val searchFragmentXML: SearchFragmentXML
)